src/workerd/api/node/process.c++

```diff
@@ -110,11 +110,11 @@ namespace {
 }  // namespace
 
 jsg::JsObject ProcessModule::getVersions(jsg::Lock& js) const {
-  auto versions = js.obj();
   // Node.js version - represents the most current Node.js version supported
   // by the platform, as defined in node-version.h
-  versions.set(js, "node"_kj, js.str(nodeVersion));
-  return versions;
+  static kj::StringPtr keys[] = {"node"_kj};
+  jsg::JsValue values[] = {js.str(nodeVersion)};
+  return js.obj(kj::ArrayPtr<kj::StringPtr>(keys), kj::ArrayPtr<jsg::JsValue>(values));
```

Comment:
can this be simply `kj::arrayPtr(values)`?

Reply:
Yes!!!!